java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -369,4 +373,69 @@ CompletableFuture<Long> linsert(
      * }</pre>
      */
     CompletableFuture<Long> lpushx(String key, String[] elements);
+
+    /**
+     * Blocks the connection until it pops and pops one or more elements from the first non-empty list
+     * from the list of provided <code>key</code> <code>BLMPOP</code> is the blocking variant of
+     * <code>LMPOP</code>.
+     *
+     * @apiNote
+     *     <ol>
+     *       <li>When in cluster mode, all <code>keys</code> must map to the same <code>hash slot
+     *           </code>.
+     *       <li><code>BLMPOP</code> is a client blocking command, see <a
+     *           href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands">Blocking
+     *           Commands</a> for more details and best practices.
+     *     </ol>
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://valkey.io/commands/blmpop/">valkey.io</a> for details.
+     * @param keys The list of provided <code>key</code> names.
+     * @param direction The direction based on which elements are popped from - see {@link
+     *     PopDirection}.
+     * @param count The count of elements to be popped and returned.
+     * @param timeout The number of seconds to wait for a blocking operation to complete. A value of
+     *     <code>0</code> will block indefinitely.
+     * @return A <code>Map</code> of <code>key</code> name mapped array of popped elements. <br>
+     *     If no member could be popped and the timeout expired, returns <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * Map<String, String[]> result = client.blmpop(new String[] {"testKey2"}, PopDirection.LEFT, 1L, 0.1).get();
+     * result.forEach((key, value) -> System.out.println("Key: " + key + ", Value: " + Arrays.toString(value)));
+     * }</pre>
+     */
+    CompletableFuture<Map<String, String[]>> blmpop(
+            String[] keys, PopDirection direction, long count, double timeout);
+
+    /**
+     * Blocks the connection until it pops and pop one element from the first non-empty list from the
+     * list of provided <code>key</code> <code>BLMPOP</code> is the blocking variant of <code>LMPOP
+     * </code>.
+     *
+     * @apiNote
+     *     <ol>
+     *       <li>When in cluster mode, all <code>keys</code> must map to the same <code>hash slot
```

Comment:
Don't ephasize hash slot

Reply:
ah, i think i was looking at the pr where bzmpop got added and it was emphasized there, but not in the lastest code, i should always reference latest down the line